static/css/all-js.less

```diff
@@ -0,0 +1,11 @@
+/**
+ * Styles here will load for all users, but only if JavaScript is enabled.
+ * At time of writing, this stylesheet is render blocking.
+ * However, the long term plan for this stylesheet is not to be render blocking.
+ */
+
+// slick font is used by Carousel arrow icons
+// (this is a lot of bytes for just an icon so should be revisited asap)
+@import (less) 'lib/font-awesome.min.css';
+@import (less) 'lib/slick.css';
+@import (less) 'lib/slick-theme.css';
```

Comment:
You'll need to add 
@import (less) 'components/backtoTop.less' here to load your CSS file.

Reply:
When I am importing here it is showing conflict please help! @jdlrobson